app/src/main/java/com/example/pocketwatching/Activities/ProfileActivity.java

```diff
@@ -199,6 +202,29 @@ public void onFailure(Call<List<TxHistory>> call, Throwable t) {
         });
     }
 
+    private void getHistoricalBalance() {
+        /*
+        1) Get the current block height from moralis
+        2) Get the block height from last week from moralis
+        3) Break block heights into intervals to get wallet balances from
+        4) Query an API for wallet balance at each interval
+        5) Add balance and time pair to another list
+        */
```

Comment:
yay, nice and neat explanatory comment! This is a minor nit, but let's move this comment right above the method signature.